docs/app/Examples/modules/Popup/Types/PopupTargetExample.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Popup, Card, Rating, Image } from 'semantic-ui-react'
+
+export default function PopupHtmlExample() {
```

Comment:
Export name doesn't match filename.  Though, this file appears the same as `PopupHtmlExample.js` and I cannot find it referenced anywhere else in this PR.  Is it cruft?